packages/backend-plugin-api/src/services/system/types.ts

```diff
@@ -38,6 +39,8 @@ export type ServiceRef<
    */
   scope: TScope;
 
+  singleton: TSingleton;
```

Comment:
I'm leaning towards the `collected?: boolean` option here, so that we can default it to `false` instead and encode that naming into the API. Happy to explore other options than "collect" though ofc

Reply:
Renamed to `multiple?: boolean`, defaults to `false`.

Reply:
Kinda leaning towards `instances?: 'single' | 'multiple'  = 'single'` with the new updates, what do you think?

Reply:
We decided to use 'singleton' and 'multiton' for now